st2client/st2client/commands/auth.py

```diff
@@ -140,6 +141,7 @@ def run(self, args, **kwargs):
 
         with open(config_file, 'w') as cfg_file_out:
             config.write(cfg_file_out)
+        os.chmod(config_file, mode=0o660)
```

Comment:
Same here. I think we previously agreed to set file permissions only on creation and don't try chmod afterwards.